prow/apis/prowjobs/v1/register.go

```diff
@@ -37,12 +37,13 @@ func Resource(resource string) schema.GroupResource {
 	return SchemeGroupVersion.WithResource(resource).GroupResource()
 }
 
+// SchemeBuilder : TODO: (alisondy) Document what SchemeBuilder is used for
```

Comment:
Please remove all comments like this. Either add the documentation or leave it undocumented.

This silences the lint error without actually addressing the intent of the lint error.